src/openforms/config/utils.py

```diff
@@ -14,3 +17,20 @@ def remove_empty_design_tokens(obj: dict) -> dict:
         result[key] = updated_value
 
     return result
+
+
+def verify_clamav_connection(host: str, port: int, timeout: int) -> dict:
```

Comment:
Can you return a dataclass instead of a dict with detailed type information, or if you really must use a dict, use a `TypedDict` so type checkers correctly check the key types?

Data class definitely has the preference though.